printer_mods/hartk1213/Voron2.4_Y_Endstop_Relocation/README.md

```diff
@@ -0,0 +1,21 @@
+This mod is for the V2 A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead and that is moving the Y endstop to the A drive this requires a 2pin jst to be soldered to the switch, 
```

Comment:
Change to relative link and fix the run-on sentence.  Here's the whole paragraph rewritten:

    This mod moves the Y endstop to the A drive, to go along with [Arkeet's MGN12 mod](../../arkeet/mgn12)
    that has the X endstop on the toolhead.  This mod requires a 2 pin JST-XH to be soldered to the
    switch.  The best way I have done it is to solder the wires to the JST-XH first, install it with a dab
    of super glue, then solder the wires to the Omron switch.